interface/forms/procedure_order/new.php

```diff
@@ -226,13 +231,27 @@ function getListOptions($list_id , $fieldnames=array('option_id', 'title', 'seq'
  padding-right:2px;
 }
 
+.form-inline
```

Comment:
@juggernautsei Though we don't have any real separation yet, "view" as I was referring to it was a file that predominantly handles displaying the UI to the user.

@bradymiller Definitely can't edit a bootstrap file directly, that has to be subclassed in SCSS by importing the main bootstrap file and then providing overrides.